### ISSUE TITLE
Heartbeat from @dmeroff

### ISSUE THREAD

**Issue by thyseus** [dektrium/yii2-user#903]
@dmeroff your project is great and alive; unfortunately you seem to have some periods of longer inactivity here and then. Please give us a sign if you think the community should fork the project and search for another maintainer.

**Comment by MKiselev** [comment]
I join you!

**Comment by thyseus** [comment]
For people interested: i have an fork of dektrium/yii2-user which has a lot of the improvements of the last months merged into. It is 43 commits ahead for the time being. It is called the 'hb' branch. https://github.com/thyseus/yii2-user/tree/hb 
@dmeroff it would be really sad if we lose you as a maintainer, you did a wonderful job with this module so far.

**Comment by philippfrenzel** [comment]
Sometimes people are more busy and sometimes they aren't - before you think about forking and building a new "trunk" - why don't you ask for teaming and maybe @dmeroff would love to share to another programmer for maintaining ? ;)

**Comment by thiagotalma** [comment]
Hello guys!  @thyseus @MKiselev @philippfrenzel 

I'm from the team but I do not have much time to evaluate everything.

If at least 3 people become PR reviewers, I'll get the approvals.

**Comment by thyseus** [comment]
I would love to get an PR reviewer / Maintainer for this project.

**Comment by thiagotalma** [comment]
We have 2 more candidates?
@SamMousa @philippfrenzel ?

**Comment by philippfrenzel** [comment]
Von meinem iPhone gesendet

**Comment by philippfrenzel** [comment]
Sorry - Mail just send without content - I can do the job for a couple of time but still would be happy if some more talented coders would step into that role - I'm kind of old school - lol

**Comment by faenir** [comment]
I would be happy to help, but I'm not sure that my knowledge will be enough.

**Comment by smilemd** [comment]
I don't have too much time but I could help from time to time.

**Comment by tsdogs** [comment]
I use it also in many projects so keeping it maintained would be great.
:+1:

**Comment by SamMousa** [comment]
@thiagotalma I can help out with reviewing.
Might be smart to separate 2 types though.
1. Code quality.
2. Direction of the project.

I'm confident I can review for quality, but I am not close enough to you guys to understand what you want in terms of future for the project.

Of course, if we review the code quality then the other one might take less time :)

**Comment by philippfrenzel** [comment]
Hi, looks like we might have enough people to step into the job - what are the next steps?

**Comment by thyseus** [comment]
In my opinion, the first step for reaching an 1.0.0 release will be to finish https://github.com/dektrium/yii2-user/issues/755 . I don´t know if @dmeroff has already started this or if need to do it 'from scratch'. @thiagotalma do you know more ?

**Comment by bscheshirwork** [comment]
I have some tests (acceptance, functional) for this module in https://github.com/bscheshirwork/yii2-app-advanced-rbac/tree/master/backend/tests and https://github.com/bscheshirwork/yii2-app-advanced-rbac/tree/master/frontend/tests

**Comment by tsdogs** [comment]
Imho before discussing on code and functionality the fork shoul be arranged.
I'd create a group on github in which create the fork an set up the users that have commit permissions (so if dmeroff will ever come back could join the development) 
Then the namespaces should be fixed accordingly and a new package on packagiest created.

I'd suggest to arrange an irc session to discuss all this (timezones permitting :) )

**Comment by thyseus** [comment]
+1 irc i am flexible

**Comment by thyseus** [comment]
Just a first random proposal: How about Tuesday, 23. May 10:00 GMT time?

**Comment by SamMousa** [comment]
That's 8am for me, but I'll be there! 8-) what channel / server

**Comment by thyseus** [comment]
Great! irc.freenode.net #yii2-user

**Comment by philippfrenzel** [comment]
If I should join I can make it ;)

Von meinem iPhone gesendet

**Comment by tsdogs** [comment]
ok, will try to be there too.

**Comment by SamMousa** [comment]
Hmm, it seems I'm a week early :D

**Comment by tsdogs** [comment]
and 4 hours too :-P
BTW I suppose that's 10:00 AM GMT right?

**Comment by thyseus** [comment]
Indeed, i tried to make a big buffer so we can collect everyone interested.

Yes AM: Tuesday, 23. May 10:00 AM GMT

**Comment by philippfrenzel** [comment]
👍  was preparing to see online and realized it's next week ... :)

**Comment by SamMousa** [comment]
@tsdogs Only now do I read your reply... Timezones, don't like 'm... See you guys in 4 hours ;-)

**Comment by tsdogs** [comment]
Just to make other people aware, we started a fork which in our wishes would only be useful to do faster development/inclusion/testing, not a replacement of this projects in the long time. [1]

Everybody is welcome to participate. 
To use the new specific repository please see [2]

[1] https://github.com/dabilite/yii2-user
[2] https://snippets.khromov.se/composer-use-your-own-fork-for-a-package/

**Comment by philippfrenzel** [comment]
Hey, why 4 Hours, thought we start in around 9 hours from now???

**Comment by thyseus** [comment]
Seems like we missed the AM/PM wrong, the irc meeting was some hours ago. We want to do it regularly, maybe once a week. @philippfrenzel we missed you :) but next time we will do it together !! Whats your opinion about our fork?

**Comment by philippfrenzel** [comment]
Hey, well I didn't miss you, as I thought it will be this evening *g* - so knowing this - I'm very sorry! So pls. next time I need a time sayin 9am CET ... lol
Anyway about the fork - I'm fine with everything that keeps code quality, respects user opinions and get's maintained! So I'll take a look at this closer this evening!  @thyseus

**Comment by tsdogs** [comment]
@philippfrenzel well that would have been 12 PM CET (midday) = 10 AM GMT (CET is UTC+2, GMT is UTC in practice) 
https://time.is/GMT
Anyway if it's ok for evetybody next meeting will be on IRC on the 30th may 2017 10:00 GMT (or UTC as u prefer), (that would be 12 PM CET (midday, NOT midnight :-) ) as most of us are on CET) 

I've opened a new Issues on the "fork" so coordination can be done there.
https://github.com/dabilite/yii2-user/issues/7

If anybody (@thiagotalma for example) wants it to be hold a bit later just let us know.

**Comment by faenir** [comment]
https://github.com/2amigos/yii2-usuario

> ## Credits
> 
> This module is highly inspired by the excellent work of [Dektrium](https://dektrium.com/) on both of its modules: 
> 
> - [Yii 2 User](https://github.com/dektrium/yii2-user)
> - [Yii 2 RBAC](https://github.com/dektrium/yii2-rbac)
> 
> In fact, we have used some of its processes, commands, views, and some of its attribute names to somehow standardize the 
> modules and make it easier for our developers to swap from [Dektrium's](https://dektrium.com) modules to our own.

**Comment by yoyhosoft** [comment]
Any update?

**Comment by bscheshirwork** [comment]
+1

**Comment by thiagotalma** [comment]
I took advantage of the holiday at the end of the year to give a work in this project.
I solved some PRs and issues.

I will do my best to keep the project running, without breakages.
But I confess that it will be difficult to implement new features without @dmeroff

This is a widely used and widespread project among Yii2 developers. I do not think the solution is to create forks and keep parallel projects.

**Comment by philippfrenzel** [comment]
@thiagotalma maybe you can add another 2 maintainers to the project to review pull requests and merge them if needed - the forking happened because of no possibility to change the master tree... I highly appreciate your work and the work from @dmeroff so keep going and yii2-user is an amazing extension!

**Comment by yoyhosoft** [comment]
Is @dmeroff still alive but he busy right?

**Comment by thyseus** [comment]
Nobody knows. His last sign of life was about 10 months ago here. We should definitely keep a look at
https://github.com/2amigos/yii2-usuario and we should try to keep contact with them. It looks really promising and could be yii2-user on steroids in the future :)

**Comment by yoyhosoft** [comment]
Yea, he still alive, I saw he keep posting pictures on Instagram lol.

**Comment by thyseus** [comment]
Probably he switched to .NET or Visual Basic </sarcasm>

**Comment by thyseus** [comment]
Or to ZEND Framework.

**Comment by thiagotalma** [comment]
>  maybe you can add another 2 maintainers to the project to review pull requests and merge them if needed

@philippfrenzel I really wanted to do this! The problem is that my user is not allowed to include members in the project. So I asked for help with the reviews for the comments.

I just got in touch with Dmitry asking him to give me permission to include new members. Let's wait....

**Comment by philippfrenzel** [comment]
@thiagotalma - understand, so @dmeroff pls. let everyone help and join sources together ! ;)

**Comment by thiagotalma** [comment]
A long time ago Dektrium added me as a member because I was helping with some PR.

But I can not assume as a maintainer, unfortunately I do not have enough time to take such responsibility.

I took advantage of my holidays at the end of the year to accept some PR that were outstanding. I also sent an email to Dektrium requesting to add me as a project admin so that I could add more members to help. Unfortunately he did not answer me.

SO, in my opinion, this project should be inactivated. Unfortunately.

As I know that it is not simple to migrate from one user manager to another, it would be important to ensure a stable version of this project. Many applications will continue to run with it.

It has already been helpful to us and deserves a worthy closing. No new features, but perfectly functional and reliable.

I would count on your help to do a final check and adjust what is pending adjustment to ensure stability.

cc: @schmunk42 @SamMousa @philippfrenzel @thyseus @bscheshirwork @tsdogs

Then I change the readme and inform that the project will be inactive.

The team @2amigos could also help in this finalization and the readme would indicate their project as successor.
cc: @amigo-tabin @dzona @jehkinen @tonydspaniard @vova07

**Comment by thyseus** [comment]
@thiagotalma I totally agree. Lets put all our efforts and power into yii2-usuario. The move over is not that complicated, because it also is a fork of dektrium/yii2-user.

**Comment by thiagotalma** [comment]
> Lets put all our efforts and power into yii2-usuario.

Okay, but first the efforts must be in order to inactivate this project with perfect working conditions. Totally stable.

**Comment by mnglkhn** [comment]
So far it seems to work fine on 7.1

**Comment by bscheshirwork** [comment]
For example: I have error in tests on page /index-test.php/user/admin/update?id=1 and 
/index-test.php/user/admin/create - don't see validation error on empty fields

**Comment by cjtterabyte** [comment]
The extension will be closed, which conclusion was reached ?, I am testing it with yii 2.1, making all the relevant changes and it works perfectly without many changes, 

cc: @schmunk42 @SamMousa @philippfrenzel @thyseus @bscheshirwork @tsdogs

Because not to make a fork with several maintainers and to start from here with Yii 2.1 and Yii 2.0 for a stable version, it really is not long, it's just my two cents.

**Comment by schmunk42** [comment]
@cjtterabyte I think we'll move over to usuario for Yii 2.1. dektirum/yii2-user hopefully will get some security fixes if needed.

**Comment by SamMousa** [comment]
I am no longer using this extension nor any other for that matter.
@schmunk42 thanks for the suggestion, I'll remember to check that out if I ever use one.

**Comment by cjtterabyte** [comment]
@schmunk42 I have advanced the use of yii 2.1., in fact I fork, which I am working with bootstrap4, and so far it is going well.

**Comment by thiagotalma** [comment]
Hi, @cjtterabyte 

The extension will not be closed nor will it die.

It will continue as a zombie, attending to those who already use it.

While the Dektrium does not wake up from its deep sleep, I try to accept the most critical PRs with the help of the guys here.

**Comment by cjtterabyte** [comment]
@thiagotalma perfect, you can then create a branch or repo new [dektrium/yii3-user] for yii 2.1. and start sending the PR, with the adjustments to be reviewed, in addition to its adaptation to Bootstrap4 you think?

**Comment by tsdogs** [comment]
This has been widely discussed (a "fork" was created in the hope that this project would then come back online, but that hasn't happened till now) 
IMHO the solution is simply to move to yii2-usuario (it is really easy and there's a very complete guide on how to do it), and we should all concentrate on the development of that. 
It has all the features and most important development is going forward there.
I've migrated many projects right now.

**Comment by thiagotalma** [comment]
@cjtterabyte , @tsdogs  is right.

**Comment by cjtterabyte** [comment]
@thiagotalma Believe me something about the work of @dmeroff and yours has been so great that works in yii 2.1. without many modifications with bootstrap4, without major problems, so I think I'll stay better here with her, see the best pr and join them in yii 2.1.

**Comment by thiagotalma** [comment]
It has an interesting PR: https://github.com/dektrium/yii2-user/pull/1061
Anyone want to help to review?

**Comment by demonking** [comment]
If it's a Zombie, like mentioned in other Issues... why it isn't present in the Readme?

**Comment by schmunk42** [comment]
I think people should move over to https://github.com/2amigos/yii2-usuario/ - we made the switch and it was rather easy.
